package-lock.json

```diff
@@ -8,14 +8,15 @@
       "name": "xplay-ott",
```

Comment:
please delete package-lock.json, we'll only keep yarn.lock as we are using yarn for package management; using two package managers may create conflicts